specification/purview/data-plane/Azure.Analytics.Purview.Account/preview/2023-10-01-preview/account.json

```diff
@@ -0,0 +1,2681 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Collections_DeleteCollection": {
+            "$ref": "./examples/Collections_DeleteCollection.json"
+          }
+        }
+      }
+    },
+    "/collections": {
+      "get": {
+        "tags": [
+          "Collection"
+        ],
+        "description": "List the collections in the account.",
+        "operationId": "Collections_ListCollections",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "query",
+            "name": "$skipToken",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CollectionList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the provider.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Collections_ListCollections": {
+            "$ref": "./examples/Collections_ListCollections.json"
+          }
+        }
+      }
+    },
+    "/collections/{collectionName}/getChildCollectionNames": {
+      "get": {
+        "tags": [
+          "Collection"
+        ],
+        "description": "Lists the child collections names in the collection.",
+        "operationId": "Collections_ListChildCollectionNames",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "collectionName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "query",
+            "name": "$skipToken",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CollectionNameResponseList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the provider.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Collections_ListChildCollectionNames": {
+            "$ref": "./examples/Collections_ListChildCollectionNames.json"
+          }
+        }
+      }
+    },
+    "/collections/{collectionName}/getCollectionPath": {
```

Comment:
This seems like it should be a POST action `"/collections/{collectionName}:getCollectionPath"`. Or better, consider to add a `select` query param to the GET collections/{collectionName} route
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#collections-query-options